ledger/complete/ledger.go

```diff
@@ -368,3 +376,26 @@ func (l *Ledger) DumpTrieAsJSON(state ledger.State, outputFilePath string) error
 
 	return trie.DumpAsJSON(writer)
 }
+
+// this operation should only be used for exporting
+func (l *Ledger) keepOnlyOneTrie(state ledger.State) error {
+	// don't write things to WALs
+	l.wal.PauseRecord()
+	defer l.wal.UnpauseRecord()
+
+	allTries, err := l.forest.GetTries()
+	if err != nil {
+		return err
+	}
+
+	targetRootHash := ledger.RootHash(state)
+	for _, trie := range allTries {
+		trieRootHash := trie.RootHash()
+		if !bytes.Equal(trieRootHash, targetRootHash) {
+			l.forest.RemoveTrie(trieRootHash)
```

Comment:
I have 2 suggestions:
(let's say `n` is the number of tries in the current forest)

1. Isn't better to create a new forest with the single trie we wanna keep, instead of removing `n-1` tries ? 

2. We could make `GetTries()` return a map of root hashes to tries, instead of a slice of tries, to get a constant lookup time. I've checked the other uses of `GetTries()` and the map return seems to work fine.